frontends/election-manager/test/render_in_app_context.tsx

```diff
@@ -122,7 +122,8 @@ export function renderInAppContext(
     <AppContext.Provider
       value={{
         castVoteRecordFiles,
-        electionDefinition,
+        electionDefinition:
+          electionDefinition === 'NONE' ? undefined : electionDefinition,
```

Comment:
I needed a way to explicitly make the `electionDefinition` `undefined` in tests since passing `undefined` causes the default to get used:

https://github.com/votingworks/vxsuite/blob/f2e6f56619d2a351c238e41605df5f08229bdbdd/frontends/election-manager/test/render_in_app_context.tsx#L87